test/extended/builds/pipeline_origin_bld_rhelimagesonly.go

```diff
@@ -0,0 +1,367 @@
+package builds
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	buildv1 "github.com/openshift/api/build/v1"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/jenkins"
+)
+
+var _ = g.Describe("[sig-builds][Feature:Builds][sig-devex][Feature:JenkinsRHELImagesOnly][Slow] openshift pipeline build", func() {
```

Comment:
Is there a reason why we want to include this in the build suite?

Reply:
good catch ... will correct